### PR TITLE
Added new migration to fixed module migrations feature.

### PR DESCRIPTION
**Pull request by pkornowski**
At first I added auto-increment in base version, but it should be placed in a separate migration. In PSQL I used sequence. I looked for solution with adding 'serial' to existing field in table, like in MySQL, but I have found only this.

### CODE REVIEW

File: platform/mds/mds/src/main/resources/db/migration/default/V6__MOTECH-2578.sql [hunk 1]
```diff
@@ -0,0 +1,22 @@
+-- Added auto increment sequence
+
```
Comment (pgesek) [inline platform/mds/mds/src/main/resources/db/migration/default/V6__MOTECH-2578.sql]: Isn't this a bit overcomplicated? Can't we just use the serial type?

**Comment by kmadej** [comment]
test this psql

File: platform/mds/mds/src/main/resources/db/migration/default/V6__MOTECH-2578.sql (hunk 1 above)
Comment (pkornowski) [inline platform/mds/mds/src/main/resources/db/migration/default/V6__MOTECH-2578.sql]: @pgesek 
I looked for solution with adding serial type. I found some solution which works, when we add a new field in the table, but when we want to change type in an existing field, it doesn't work.

**Comment by pkornowski** [comment]
test this psql

File: platform/mds/mds/src/main/resources/db/migration/default/V6__MOTECH-2578.sql [hunk 2]
```diff
@@ -0,0 +1,5 @@
+-- Added auto increment sequence
```
Comment (kmadej) [inline platform/mds/mds/src/main/resources/db/migration/default/V6__MOTECH-2578.sql]: Changed primary key to serial type

**Comment by pkornowski** [comment]
@pgesek 
Can we merge this?

**Comment by kmadej** [comment]
@pgesek 
Looks good to me, can we merge this ?

**Pull request merged**